setup.py

```diff
@@ -7,6 +7,6 @@
       packages=find_packages(exclude=('docs')),
       install_requires=[
           'requests==2.10.0',
-          'pypif==1.0.24',
+          'pypif',
```

Comment:
> It is not considered best practice to use install_requires to pin dependencies to specific versions, or to specify sub-dependencies (i.e. dependencies of your dependencies). This is overly-restrictive, and prevents the user from gaining the benefit of dependency upgrades.

From https://packaging.python.org/requirements/#install-requires

If we stick to semantic versioning, we can specify upper and lower bounds to reflect sensitivity to the API.